docker-tags/action.yml

```diff
@@ -45,5 +46,5 @@ runs:
         echo "PARAM_DRYRUN=${{ inputs.dry-run }}" >> $GITHUB_ENV
     - name: Run script
       shell: bash
-      run: ./scripts/docker-tags.sh
+      run: bash ${{ github.workspace }}/../../_actions/jahia/jahia-modules-action/TECH-532/docker-tags/scripts/docker-tags.sh
```

Comment:
You should replace `${{ github.workspace }}/../../_actions/jahia/jahia-modules-action/TECH-532/docker-tags` 

with `${{ env.GITHUB_ACTION_PATH}}` or `${{ github.action_path }}`. I'm not sure how it deals with path though, so you'd need to investigate a bit.